cetmix_tower_server/tests/test_plan.py

```diff
@@ -1583,6 +1583,8 @@ def test_plan_line_action_variable_values_user_access_rights(self):
         #  plan line action
         # Add user to group
         self.add_to_group(self.user_bob, "cetmix_tower_server.group_user")
+        self.variable_os.write({"access_level": "1"})
+        self.variable_value.write({"access_level": "1"})
         test_plan_line_action_as_bob = self.plan_3_line_1_action.with_user(
```

Comment:
This is a computed field that should be updated automatically when the variable access level is updated.